we_count/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    node: true
+  },
+  parserOptions: {
+    parser: 'babel-eslint'
+  },
+  extends: [
+    '@nuxtjs',
+    'plugin:nuxt/recommended'
```

Comment:
We have a common [eslint config](https://github.com/fluid-project/eslint-config-fluid) for our javascript projects. What are the main differences and is there a reason to be using the one supplied by nuxt over the fluid version?

Reply:
Not at all. It was the default linter configuration. Will have it changed for the next pull request.